package.json

```diff
@@ -65,7 +65,7 @@
     "arbitrary": "^1.4.10",
     "base58check": "^2.0.0",
     "bcash": "^1.1.1",
-    "bcoin": "^1.0.2",
+    "bcoin": "bcoin-org/bcoin#master",
```

Comment:
This was changed as `bcoin@1.0.2` used an incorrect prefix for `regtest` networks, but also broke browser functionality.